### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by JahongirmirzoDv** [bumble-tech/appyx#46] [Q&A] [answered]
it gives such an error

<img width="1440" alt="Screen Shot 2022-07-30 at 13 33 45" src="https://user-images.githubusercontent.com/74462457/181900851-1fdeaf05-c799-4a6d-ad52-0d94ad26c636.png">
<img width="1440" alt="Screen Shot 2022-07-30 at 13 33 39" src="https://user-images.githubusercontent.com/74462457/181900904-8e389661-3b9b-40a6-a2ce-afe6a3758d3c.png">
<img width="1440" alt="Screen Shot 2022-07-30 at 13 32 56" src="https://user-images.githubusercontent.com/74462457/181900929-e0feca2f-4e5b-4a99-bc5a-0b618becabd4.png">
r

**Comment by KovalevAndrey** [comment, marked as answer]
Hi @JahongirmirzoDv you're missing `androidx.appcompat:appcompat:x.y.x` dependency which we should probably expose as API dependency for the clients.

**Reply by JahongirmirzoDv** [reply]
Thank you it worked
